commands/command_clean.go

```diff
@@ -77,8 +77,9 @@ func cleanCommand(cmd *cobra.Command, args []string) {
 func init() {
 	RegisterSubcommand(func() *cobra.Command {
 		return &cobra.Command{
-			Use: "clean",
-			Run: cleanCommand,
+			Use:    "clean",
+			Run:    cleanCommand,
+			PreRun: resolveLocalStorage,
```

Comment:
I wonder if we could remove all the duplication of setting `PreRun` (and potentially forgetting to  include it) by standardising on a wrapper function like `NewLfsCommand("clean", cleanCommand)` or similar?